app/src/main/java/com/example/parkingsystemjava/mvp/view/ParkingView.java

```diff
@@ -0,0 +1,25 @@
+package com.example.parkingsystemjava.mvp.view;
+
+import android.app.Activity;
+import android.content.Context;
+import android.widget.Toast;
+import com.example.parkingsystemjava.R;
+import com.example.parkingsystemjava.databinding.ActivityMainBinding;
+import com.example.parkingsystemjava.mvp.contract.ParkingContract;
+
+public class ParkingView extends ActivityView implements ParkingContract.ViewContract {
+
+    private final ActivityMainBinding binding;
+
+    public ParkingView(Activity activity, ActivityMainBinding binding) {
+        super(activity);
+        this.binding = binding;
+    }
+
+    @Override
+    public void showParkingsAvailable(int parkingsAvailable) {
+        Context context = getContext();
+        Toast.makeText(context, context.getString(R.string.toast_main_parking_size, parkingsAvailable), Toast.LENGTH_LONG).show();
```

Comment:
please, check if the context is not null before use it

Reply:
done